AxonDeepSeg/ads_utils.py

```diff
@@ -43,9 +43,11 @@ def _main_thread_terminated(self):
                 print("Press Ctrl-C to quit")
 
             # -- Function override statement --#
+            configPath = getConfigPath()
```

Comment:
You don't have to change this but take a look on PEP8 for naming conventions. The convention for function names is like `get_config_path()` and for the variable it should be like `config_path`.